providers/scaleway/doc.go

```diff
@@ -0,0 +1,14 @@
+// Package scaleway implements a way to use the Scaleway Cloud Provider for your
```

Comment:
Not sure about this one. As we don't use such a pattern anywhere in the woodpecker org, I would prefer to remove it. @anbraten do you have an opinion regarding this?

Reply:
I can remove it, but that's a standard used in Go to centralise the `overview` documentation of a package in a code unit.

e.g. https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/doc.go
gives https://pkg.go.dev/k8s.io/kubernetes/pkg/kubelet#pkg-overview

Reply:
Im ok with it if nobody else has another opinion.